lib/components/FileAttachment/Attachment.jsx

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from "react";
+
+import { MenuVertical, File } from "neetoicons";
+import { Dropdown, Input, Typography } from "neetoui";
+
+import { ATTACHMENT_OPTIONS } from "./constants";
+
+const { Menu, MenuItem } = Dropdown;
+
+const Attachment = ({ attachment, dropDownOptions = [] }) => {
+  const [activeFile, setActiveFile] = useState({});
```

Comment:
The `Attachment` component will always keep track of a single file. Hence, this state does not make any sense. Please rethink the logic/state names.